packages/viem/package.json

```diff
@@ -21,25 +21,13 @@
     "gen:abis": "pnpm dlx tsx ./scripts/abigen",
     "gen:chains": "pnpm dlx tsx ./scripts/chaingen",
     "gen:abi-docs": "pnpm dlx tsx ./scripts/abidocgen",
-    "lint": "eslint \"**/*.{ts,tsx}\" && pnpm prettier --check \"**/*.{ts,tsx}\"",
-    "lint:ci": "eslint \"**/*.{ts,tsx}\" --quiet && pnpm prettier --check \"**/*.{ts,tsx}\"",
-    "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix --quiet && pnpm prettier \"**/*.{ts,tsx}\" --write --loglevel=warn",
-    "test": "vitest",
-    "test:ci": "vitest --run",
-    "typecheck": "tsc --noEmit --emitDeclarationOnly false",
-    "typecheck:ci": "tsc --noEmit --emitDeclarationOnly false"
+    "lint": "eslint \"**/*.{ts,tsx}\" && pnpm prettier --check \"**/*.{ts,tsx}\" --ignore-path \"../../.prettierignore\"",
+    "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix && pnpm prettier \"**/*.{ts,tsx}\" --write --loglevel=warn --ignore-path \"../../.prettierignore\"",
+    "test": "vitest --run",
+    "typecheck": "tsc --noEmit --emitDeclarationOnly false"
   },
-  "devDependencies": {
-    "@types/node": "^22.5.4",
-    "@viem/anvil": "^0.0.7",
-    "@iarna/toml": "^2.2.5",
-    "eta": "^3.1.1",
-    "resolve-tspaths": "^0.8.18",
-    "typedoc": "^0.26.7",
-    "typedoc-plugin-markdown": "^4.2.8",
-    "typescript": "^5.2.2",
-    "viem": "^2.17.9",
-    "vitest": "^1.6.0"
+  "dependencies": {
+    "viem": "^2.17.9"
```

Comment:
similar comment here, I dont think we want to enforce a specific version of viem as a dependency

Reply:
we have to since this package uses viem as a dependency

Reply:
actually for this we should be able to leverage peerDependencies. This is different than the devDependencies. Let me revisit it

Reply:
updated! good catch